src/olympia/abuse/cinder.py

```diff
@@ -294,16 +326,18 @@ def __init__(self, collection):
 
     @property
     def id(self):
-        return str(self.collection.id)
+        return self.get_str(self.collection.id)
 
     def get_attributes(self):
+        # FIXME: translate translated fields in reporter's locale, send as
+        # dictionaries?
         return {
             'id': self.id,
-            'slug': self.collection.slug,
-            # FIXME: locales!
-            'name': str(self.collection.name),
-            'description': str(self.collection.description),
             'comments': self.collection.get_all_comments(),
+            'description': self.get_str(self.collection.description),
+            'modified': self.get_str(self.collection.modified),
+            'name': self.get_str(self.collection.name),
+            'slug': self.collection.slug,
         }
```

Comment:
I'm voting no for the moment because it's really not clear to me whether or not this is a good idea, it could be misleading (like a moderator deleting a collection because it contains a problematic add-on - that the author could have modified without the collection owner noticing!). Needs more discussion and a decision on the proper approach.